assets/settings/default.json

```diff
@@ -138,6 +138,12 @@
     // Whether to show diagnostic indicators in the scrollbar.
     "diagnostics": true
   },
+  "gutter": {
+    // Whether to show line numbers in the gutter.
+    "line_numbers": true,
+    // Whether to show code action/folding buttons in the gutter.
+    "buttons": true
```

Comment:
I think this would be clearer for users (And in the implementation) if it was two settings:
* "code_actions": true,
* "folds": true,

Reply:
I added these options and also made the gutter padding calculation a bit clearer (I split the padding into left/right instead of just one variable).